pkg/polarion-xml/polarion_xml.go

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package polarion_xml
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+)
+
+type TestCases struct {
+	XMLName   xml.Name   `xml:"testcases"`
+	TestCases []TestCase `xml:"testcase"`
+	ProjectID string     `xml:"project-id,attr"`
+}
+
+type TestCase struct {
+	Title                Title                `xml:"title"`
+	Description          Description          `xml:"description"`
+	TestCaseCustomFields TestCaseCustomFields `xml:"custom-fields"`
+	TestCaseSteps        TestCaseSteps        `xml:"test-steps"`
+}
+
+type Title struct {
+	Content string `xml:",chardata"`
+}
+
+type Description struct {
+	Content string `xml:",chardata"`
+}
+
+type TestCaseCustomFields struct {
+	CustomFields []TestCaseCustomField `xml:"custom-field"`
+}
+
+type TestCaseCustomField struct {
+	Content string `xml:"content,attr"`
+	ID      string `xml:"id,attr"`
+}
+
+type TestCaseSteps struct {
+	Steps []TestCaseStep `xml:"test-step"`
+}
+
+type TestCaseStep struct {
+	StepColumn []TestCaseStepColumn `xml:"test-step-column"`
```

Comment:
it will affect XML generation, current XML format must be
```
<testcase assignee-id="testuser2" id="shotwell_customid_1" due-date="2016-09-30" initial-estimate="">
    <title>Import and Display jpg files </title>
    <description>import and display jpg files</description>
    <test-steps>
      <test-step>
        <test-step-column id="expectedResult">Files can be seen and can be displayed.</test-step-column>
        <test-step-column id="step">import jpg files into Shotwell</test-step-column>
      </test-step>
    </test-steps>
    <custom-fields>
      <custom-field content="notautomated" id="caseautomation"/>
      <custom-field content="medium" id="caseimportance"/>
      <custom-field content="component" id="caselevel"/>
      <custom-field content="positive" id="caseposneg"/>
      <custom-field content="yes" id="upstream"/>
      <custom-field content="functional" id="testtype"/>
      <custom-field content="-" id="subtype1"/>
      <custom-field content="-" id="subtype2"/>
      <custom-field content="tags" id="tags"/>
      <custom-field content="setup" id="setup"/>
      <custom-field content="teardown by cleaning the workspace." id="teardown"/>
      <custom-field content="
http://file.bos.redhat.com/~username/myscript.sh
" id="automation_script"/>

    </custom-fields>
  </testcase>
```